Changelog.md

```diff
@@ -4,6 +4,8 @@ Language Features:
 
 
 Compiler Features:
+ * Assembler: Use ``push0`` for placing ``0`` in the stack for EVM versions starting from "Shanghai". This decreases the deployment costs.
```

Comment:
This decreases runtime cost too, 3 gas vs 2 gas.

Reply:
Will add it.